src/libs/actions/Policy.ts

```diff
@@ -660,28 +724,37 @@ function updateWorkspaceAvatar(policyID, file) {
             },
         },
     ];
-    const failureData = [
+    const failureData: OnyxUpdate[] = [
         {
             onyxMethod: Onyx.METHOD.MERGE,
             key: `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
             value: {
-                avatar: allPolicies[`${ONYXKEYS.COLLECTION.POLICY}${policyID}`].avatar,
+                avatar: allPolicies?.[`${ONYXKEYS.COLLECTION.POLICY}${policyID}`]?.avatar,
                 pendingFields: {
```

Comment:
This looks like a case where the `pendingFields` can be included in `finallyData` to DRY this up and then the difference with `avatarData` can be alone in `failureData`